### PR TITLE
Add moar tests

### PR DESCRIPTION
**Pull request by johnsonm325**

### CODE REVIEW

File: src/SmartComponents/__tests__/completedTaskDetailsHelpers.tests.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
+import {
+  getSelectedSystems,
+  isError,
+  fetchTask,
+  fetchTaskJobs,
+} from '../completedTaskDetailsHelpers';
+import { log4j_task_jobs } from '../CompletedTaskDetails/__tests__/__fixtures__/completedTasksDetails.fixtures';
+import {
+  errorResponse,
```
Comment (leSamo) [inline src/SmartComponents/__tests__/completedTaskDetailsHelpers.tests.js:9]:
```
export const isError = (result) => {
  return result?.response?.status && result?.response?.status !== 200;
};
```

201, 202, 203, 204, 205, 206 response codes be like:
Untitled-5

I know the API probably uses 200 only, but let's futureproof this by checking for the first digit :smile:

File: src/SmartComponents/__tests__/completedTaskDetailsHelpers.tests.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  });
+});
+
+describe('fetchTask', () => {
+  const setError = jest.fn();
+
+  it('should return fetched data', async () => {
+    fetchExecutedTask.mockImplementation(async () => {
+      return {
+        response: { status: 200 },
```
Comment (leSamo) [inline src/SmartComponents/__tests__/completedTaskDetailsHelpers.tests.js:47]: You created and tested these nice `successResponse` and `errorResponse` constants and you're not using them everywhere :smile: It does not save any lines of code, but might avoid a future confusion when a dev edits those constants and changes reflect only somewhere. Maybe also use upper case for constants, but this is super nitpicky.

**Review by leSamo — COMMENTED** [review]
Tests look good, just have some nitpicky comments.

**Comment by johnsonm325** [comment]
@leSamo you can take another look at this. Did a little "refactoring".

**Review by leSamo — APPROVED** [review]

**Pull request merged**